include/config.h

```diff
@@ -203,3 +208,9 @@
 //#define UNLOCK_ALL
 
 // If you want to change the extended boundaries mode, go to engine/extended_bounds.h and change EXTENDED_BOUNDS_MODE
+
+// -- Compatibility safeguards. Don't mess with these unless you know what you're doing.--
+#ifndef KEEP_MARIO_HEAD
+#undef GODDARD_EASTER_EGG
+#define DISABLE_DEMO
+#endif
```

Comment:
maybe these could be assertions? then a compiler error could help people understand the incompatibility of the options that they chose

Reply:
non blocking